README.md

```diff
@@ -128,16 +109,3 @@ When a bug or feature is identified which requires a change.
 10. Change request for anything big (in accordance with ITIL principles)
 11. Deploy/update component in 'prod' environment
 
-**Writing and running [Ansible](Ansible/README.md) playbooks is the recommended practice for deployment of applications.**
```

Comment:
Maybe can leave this? Might be still good to know for trying to understand production/staging 

(also this folder/file still exists, if the intention was to delete it entirely like you did for jenkins/vagrant?)